apps/site/src/views/Home/sections/FAQ/FAQ.module.scss

```diff
@@ -0,0 +1,121 @@
+@use "zothacks-theme" as theme;
+@use "bootstrap-utils" as utils;
+
+.faq {
+	position: relative;
+	display: flex;
+	flex-direction: column;
+
+	margin-left: 40px;
+	margin-right: 40px;
+	padding-bottom: 30px;
+
+	background-color: theme.$beige;
+}
+
+.title {
+	align-self: center;
+	margin-top: 24px;
+	margin-bottom: 24px;
+}
+
+.vertical-line {
+	position: absolute;
+	justify-self: flex-start;
+
+	height: 100%;
+	width: 116px;
+	border-right: 4px solid theme.$red;
+	opacity: 0.45;
+}
+
+.body {
+	margin-left: 110px;
+	margin-right: 10px;
+}
+
+.accordion {
+	--bs-accordion-bg: transparent;
+	--bs-accordion-active-bg: transparent;
+	--bs-accordion-border-color: transparent;
+	--bs-accordion-btn-focus-box-shadow: 0;
+	--bs-accordion-border-radius: 0;
+	--bs-accordion-btn-icon-width: 14px;
+}
+
+.accordion-border {
+	border-bottom: 3px solid theme.$light-blue;
+	border-top: 3px solid theme.$light-blue;
+}
+
+.star-lg {
+	position: absolute;
+	left: -60px;
+	top: -120px;
+	transform: rotate(-25deg);
+	filter: drop-shadow(0px 4px 2px rgba(0, 0, 0, 0.25));
+
+	width: 200px;
+	height: 200px;
+}
+
+.star-sm {
+	position: absolute;
+	right: -30px;
+	top: -60px;
+	transform: rotate(15deg);
+	filter: drop-shadow(0px 4px 2px rgba(0, 0, 0, 0.25));
+
+	width: 120px;
+	height: 120px;
+	visibility: hidden;
+}
+
+.eraser {
+	position: absolute;
+	left: -80px;
+	bottom: -110px;
+	filter: drop-shadow(0px 4px 2px rgba(0, 0, 0, 0.25));
+
+	width: 220px;
+	height: 220px;
+	visibility: hidden;
+}
+
+@include utils.media-breakpoint-down(md) {
+	.body {
+		margin-left: 60px;
+	}
+
+	.vertical_line {
+		width: 70px;
+	}
+}
+
+@include utils.media-breakpoint-down(sm) {
+	.body {
+		margin-left: 40px;
+	}
+
+	.vertical-line {
+		border-right: 3px solid theme.$red;
+		width: 45px;
+	}
+
+	.accordion-border {
+		border-bottom: 2px solid theme.$light-blue;
+		border-top: 2px solid theme.$light-blue;
+	}
```

Comment:
Chore: we can specify just the portion of the border that needs to be altered (e.g. `border-right-width`) instead of the entire border specification.

Reply:
hi, would you want me to split each line into two corresponding to width and color?

Reply:
Since color is already specified in the base, you can alter just the width, something like this:
```scss
.vertical-line {
	border-right: 3px solid rgba(theme.$red, 0.45);
}

.accordion-border {
	border-bottom: 2px solid theme.$light-blue;
	border-top: 2px solid theme.$light-blue;
}

@include utils.media-breakpoint-up(sm) {
	.vertical-line {
		border-width: 4px;
	}

	.accordion-border {
		border-bottom-width: 3px;
		border-top-width: 3px;
	}
}
```